### ISSUE TITLE
Cannot find GetMaxGflopsDeviceId() method in CudaContext class

### ISSUE THREAD

**Issue by vitaliizabolotnyi** [kunzmi/managedCuda#93]
Hi, I cannot find GetMaxGflopsDeviceId() method in CudaContext class. Is it removed?

**Comment by kunzmi** [comment]
Yes, it has been removed quite a while ago, please see my last answer [here](https://github.com/kunzmi/managedCuda/issues/44) for why and how to replace it.

Cheers, 
Michael
